### PR TITLE
Create 3.cn.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

File: src/chrome/content/rules/3.cn.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	
+-->
+
+<ruleset name="3.cn (partial)">
+
+	<target host="ad.3.cn" />
+	<target host="dc.3.cn" />
+	<target host="p.3.cn" />
+	<target host="pi.3.cn" />
+	<target host="yx.3.cn" />
```
Comment (J0WI) [inline src/chrome/content/rules/3.cn.xml]: I get a mismatch here

**Comment by J0WI** [comment]
I wonder why all tests passed on the mismatched host @Hainish?
https://www.ssllabs.com/ssltest/analyze.html?d=yx.3.cn

**Comment by renyouguo** [comment]
@J0WI It is ok for me.
Maybe the sslabs's error?

**Comment by J0WI** [comment]
Same with curl:
```
$ curl -I https://yx.3.cn
curl: (51) SSL: no alternative certificate subject name matches target host name 'yx.3.cn'
```

**Comment by Hainish** [comment]
This test failed with my local docker test, which mirrors the travis test:

```shell
(.venv)user@https-everywhere ~/workspace/https-everywhere (renyouguo-patch-142) $ sudo docker run --rm -ti -v $(pwd):/opt -e FIREFOX=/$FIREFOX/firefox/firefox --privileged httpse bash -c "service miredo start && test/travis-ruleset-fetch.sh"
 * Starting Teredo IPv6 tunneling daemon miredo                                                                                                                                                             [ OK ] 
From https://github.com/EFForg/https-everywhere
 * branch            master     -> FETCH_HEAD
 * [new branch]      master     -> upstream-for-travis/master
Ruleset database has changed. Testing test URLs in all changed rulesets.
INFO Finished comparing http://dc.3.cn/cathot/get2?callback=jQuery3666891&_=1466600720701 -> https://dc.3.cn/cathot/get2?callback=jQuery3666891&_=1466600720701. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://c.3.cn/quality?callback=jQuery2109499&skuId=10309761430&cat=9987%2C653%2C655&brand=12669&venderId=124424&_=1466600920610 -> https://c.3.cn/quality?callback=jQuery2109499&skuId=10309761430&cat=9987%2C653%2C655&brand=12669&venderId=124424&_=1466600920610. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://c0.3.cn/stock?skuId=10309761430&venderId=124424&cat=9987,653,655&area=1_72_2799_0&buyNum=1&extraParam={%22originid%22:%221%22}&ch=1&pduid=1292621468&pdpin=&callback=getStockCallback -> https://c0.3.cn/stock?skuId=10309761430&venderId=124424&cat=9987,653,655&area=1_72_2799_0&buyNum=1&extraParam={%22originid%22:%221%22}&ch=1&pduid=1292621468&pdpin=&callback=getStockCallback. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://d.3.cn/desc/10055355038?cdn=2&callback=showdesc -> https://d.3.cn/desc/10055355038?cdn=2&callback=showdesc. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://dc.3.cn/cathot/get2?callback=jQuery3666891&_=1466600720701 -> https://dc.3.cn/cathot/get2?callback=jQuery3666891&_=1466600720701. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://p.3.cn/prices/get?type=1&area=1_72_2799&pdtk=&pduid=1292621468&pdpin=&pdbp=0&skuid=J_10309761430&callback=cnp -> https://p.3.cn/prices/get?type=1&area=1_72_2799&pdtk=&pduid=1292621468&pdpin=&pdbp=0&skuid=J_10309761430&callback=cnp. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://yx.3.cn/service/info.action?callback=_initLite&id=1253759536&area=2_2824_0&u_source=qq&t=0.15787275404425993 -> https://yx.3.cn/service/info.action?callback=_initLite&id=1253759536&area=2_2824_0&u_source=qq&t=0.15787275404425993. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://ad.3.cn/ -> https://ad.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://c.3.cn/ -> https://c.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://c0.3.cn/ -> https://c0.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://d.3.cn/ -> https://d.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://dc.3.cn/ -> https://dc.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://p.3.cn/ -> https://p.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://pi.3.cn/ -> https://pi.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
INFO Finished comparing http://yx.3.cn/ -> https://yx.3.cn/. Rulefile: src/chrome/content/rules/3.cn.xml.
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://c.3.cn/quality?callback=jQuery2109499&skuId=10309761430&cat=9987%2C653%2C655&brand=12669&venderId=124424&_=1466600920610 => https://c.3.cn/quality?callback=jQuery2109499&skuId=10309761430&cat=9987%2C653%2C655&brand=12669&venderId=124424&_=1466600920610: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'c.3.cn'")
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://c0.3.cn/stock?skuId=10309761430&venderId=124424&cat=9987,653,655&area=1_72_2799_0&buyNum=1&extraParam={%22originid%22:%221%22}&ch=1&pduid=1292621468&pdpin=&callback=getStockCallback => https://c0.3.cn/stock?skuId=10309761430&venderId=124424&cat=9987,653,655&area=1_72_2799_0&buyNum=1&extraParam={%22originid%22:%221%22}&ch=1&pduid=1292621468&pdpin=&callback=getStockCallback: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'c0.3.cn'")
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://yx.3.cn/service/info.action?callback=_initLite&id=1253759536&area=2_2824_0&u_source=qq&t=0.15787275404425993 => https://yx.3.cn/service/info.action?callback=_initLite&id=1253759536&area=2_2824_0&u_source=qq&t=0.15787275404425993: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'yx.3.cn'")
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://c.3.cn/ => https://c.3.cn/: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'c.3.cn'")
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://c0.3.cn/ => https://c0.3.cn/: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'c0.3.cn'")
ERROR src/chrome/content/rules/3.cn.xml: Fetch error: http://yx.3.cn/ => https://yx.3.cn/: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'yx.3.cn'")
INFO Finished in 99.72 seconds. Loaded rulesets: 1, URL pairs: 15.
Test URL test failed.
```

Re-running on travis now.

**Comment by renyouguo** [comment]
3.cn belong the jd.com

**Comment by Hainish** [comment]
Super strange, the test passed again from Travis.  I wonder if this is a network perspective issue?

**Comment by Hainish** [comment]
Doesn't seem to be a network perspective issue.  On travis, this fails as expected: https://travis-ci.org/Hainish/gnutls-poc/builds/140116928

**Comment by renyouguo** [comment]
@J0WI @Hainish 

> pycurl.error: (51, "SSL: certificate subject name (*.jd.com) does not match target host name 'yx.3.cn'")
This is not an issue. As I have said, 3.cn is an important host of jd.com ,just like githubusercontent.com for github.

**Comment by renyouguo** [comment]
@J0WI @Hainish Same issue: #5003

**Comment by J0WI** [comment]
Tests passed now, for whatever reason...

**Pull request merged**